app/experimenter/docs/adrs/0001-create-new-nimbus-application.md

```diff
@@ -0,0 +1,119 @@
+# Create A New Web Application To Manage Nimbus Experiments
+
+* Status: proposed
+* Deciders: Kate Hudson, Jared Lockhart, TBD
+* Date: 2020-08-18
+
+## Context and Problem Statement
+
+Experimenter is a web application developed solely for the purpose of managing product experiments at Mozilla.  It was originally designed to manage the Shield experimentation workflow, and has expanded to include a comprehensive workflow for managing Normandy experiments.  Project Nimbus is being developed with very different core assumptions than the previous experimentation platforms, and it is unclear whether building on top of the existing Experimenter infrastructure or creating a new purpose specific application will best suit the project's needs.
+
+## Decision Drivers <!-- optional -->
+
+* Meeting Project Nimbus' needs in a rapid, continuous fashion with minimal downtime
+* Contributability by a large, distributed group of people with different backgrounds (web engineers, data engineers, data scientists)
+* Fitting cohesively into the FXA engineering pipeline/culture
+* Continuing to support the existing experimentation (Normandy) workflow
```

Comment:
I suspect that supporting Normandy existing workflow is 12-18 months at least, with fewer use cases going through Normandy as we transition more to the Nimbus flow.  We all hope it will be quicker - but just based on past experience - removing humans can mean lots of supporting work to ensure that is smooth.

It's not just normandy pref-flip experiments that experimenter supports.  It's also rollouts and add-ons.  It's also the pref-flips used to experiment on other systems without built in experimentation (ex: pocket, snippets, surveys). 

Nimbus will start taking over feature flags and xman direct integration - but there are more integration efforts on the path to Nimbus replacement for experimentation.  

The deprecation or re-homing investigation effort will continue in parallel (for capabilities we don't plan to move to Nimbus).